tests/src/test_helpers/ssl_helpers.c

```diff
@@ -12,9 +12,7 @@
 #include "md_psa.h"
 
 #if defined(MBEDTLS_SSL_TLS_C)
-#if defined(MBEDTLS_SSL_HANDSHAKE_WITH_CERT_ENABLED)
-static int rng_seed = 0xBEEF;
-static int rng_get(void *p_rng, unsigned char *output, size_t output_len)
+int mbedtls_test_random(void *p_rng, unsigned char *output, size_t output_len)
 {
     (void) p_rng;
     for (size_t i = 0; i < output_len; i++) {
```

Comment:
Pre-existing: the fact that this file calls `srand()` and `rand()` causes some static analysis tools to ~have kittens~ complain.  Maybe we should just implement our own simple PRNG within this file? It's only a couple of lines of code, and doesn't really have any requirements as to the quality of the randomness...

Reply:
Feel out of scope to me. Not done yet.